bpf/enhancedrecording/command.bpf.c

```diff
@@ -24,6 +26,15 @@ enum event_type {
     EVENT_RET,
 };
 
+// common_data_t is a struct used to store fields that are common across
+// multiple events. Those fields are the same as `data_t`.
+struct common_data_t {
```

Comment:
nit: is it necessary to define a new struct?

Reply:
This is how to avoid passing all information as function arguments or reading trace information for every event produced (per argument). We cannot use the same struct because the `argv` attribute is too large to be kept in stack memory.